### PR TITLE
Fix for drive.google.com

### PR DESCRIPTION
**Pull request by Elaborendum**
- Lines that separate folders in the path to a file/folder
- Icons in "add shortcut to Drive" option.
- Dialog box symbol that appears if no file is selected.

### CODE REVIEW

**Review by Gusted — CHANGES_REQUESTED** [review]
The usage of jiberish CSS is not recommended, especially on this site as I've seen it changed multiple times a day. Everything must be done trough more generic attribute selectors.

**Comment by Elaborendum** [comment]
I hope the changes I made aren't too broad either.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -3684,6 +3684,9 @@ div[role="menuitem"] svg
 div[role="dialog"][style^="width: 350px;"] div[role="button"][tabindex="0"]
 div[role="dialog"] ~ div[role="menu"] > div[role="menuitem"] > div > div:not([style*="background-image"])
 img[src$="type/audio/mp3"]
+div[role="listitem"] > div > div > div > svg
+div[role="menuitem"] > div > div > div
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3688]: Only this one concerns me as every "right-click menu" has such class and will definitely have more than 3 div's inside of it. Do you have an example wherefor this is used?

Comment (Elaborendum) [reply src/config/dynamic-theme-fixes.config:3688]: imagen
I don't exactly understand what you mean, but Line 3688 is for the icons from the "add shortcut to drive" menu that appears once you select the option from the main right-click menu. I tried, but I don't think it is within my ability to narrow this one down further.

On the other hand, would changing Line 3687 to `div[role="listitem"] > div > div > div > svg[fill="#000000"] > path` be of any help in narrowing that one down?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:3688]: The 2 other lines don't concern me for now, but ` svg[fill="#000000"] > path` would be helpful.

Welp, we actually encounter such `invert` problem from the drive.google.com/file viewer and with a slightly modified CSS the same trick will work
```CSS
div[role="menu"] > div > div[role="menuitem"] > div > div > div
div[role="menu"] > div > div[role="menuitem"][class*=" "] > div > :last-child
```

**Review by Gusted — APPROVED** [review]

**Comment by Vedanth-P** [comment]
Hello,
Sorry for bringing up this issue again, but I think I'm encountering problems with a line of code that was added in this pull request.

I am seeing that elements that are selected by ```div[role="menu"] > div > div[role="menuitem"][class*=" "] > div > :last-child``` are being wrongly inverted. Perhaps it was different when this pull request was merged, but currently each item on the right-click menus has an image, beside which is text, beside which (sometimes) is a second image. On menu items that do not have this second image, the selector targets the text, and it gets inverted.
Menu Item without Image Menu Item with Picture

So I removed the ```div[role="menu"] > div > div[role="menuitem"][class*=" "] > div > :last-child``` selector from the ```INVERT``` list, and the icons and dividers (which I understand are what this pull request aimed to rectify) seem to look fine. In addition, some of the icons were in fact wrongly inverted (e.g. the arrow on the right of the "Open with" menu item), but they seem fine when I remove the selector.
Wrong Inversion
Corrected Inversion


I just want to clarify that I am not missing anything, before going ahead and submitting this as a fix.

**Pull request merged**